docs/2.develop/contracts/environment/environment.md

```diff
@@ -98,47 +94,47 @@ The NEAR blockchain groups blocks in [Epochs](broken). The `current_epoch` attri
 
 ### Block Index
 
-The `block_index` represents the number of the block in which this transaction will be added to the blockchain
+The `block_index` represents the index of the block in which this transaction will be added to the blockchain
 
 ---
 
 ## Gas
 
-Your smart contract has a limited number of computational resources to use each on each call. Such resources are measured in [GAS](broken). Basically, each code instruction cost a certain amount of GAS, and if you ran out of it, the execution halts with the error message `Exceeded the prepaid gas`.
+Your smart contract has a limited number of computational resources to use on each call. Such resources are measured in [Gas](broken). Gas can be thought of as wall compute time. 1 PetaGas (1_000 TGas) is ~1 second of compute time. Each code instruction costs a certain amount of Gas, and if you run out of it, the execution halts with the error message `Exceeded the prepaid gas`.
```

Comment:
Gas can be thought of as wall time, where 1 PetaGas (1,000 TGas) is ~1 second of compute time.